matrix-sdk-android/src/main/java/im/vector/matrix/android/internal/session/group/DefaultGetGroupDataTask.kt

```diff
@@ -53,12 +54,12 @@ internal class DefaultGetGroupDataTask @Inject constructor(
         insertInDb(groupSummary, groupRooms, groupUsers, groupId)
     }
 
-    private fun insertInDb(groupSummary: GroupSummaryResponse,
+    private suspend fun insertInDb(groupSummary: GroupSummaryResponse,
                            groupRooms: GroupRooms,
                            groupUsers: GroupUsers,
                            groupId: String) {
         monarchy
-                .writeAsync { realm ->
+                .awaitTransaction { realm ->
```

Comment:
Wasn't it better to write asynchronuously?

Reply:
In tasks we should always use awaitTransaction (which is kind of async as it's a suspending method)